packages/react/src/views/AttachmentHandler/Attachment.js

```diff
@@ -14,6 +14,8 @@ const Attachment = ({ attachment, host, type, variantStyles = {} }) => {
         attachment={attachment}
         host={host}
         variantStyles={variantStyles}
+        authorIcon={attachment.author_icon}
```

Comment:
Send only one prop, i.e., `author` or `authorInfo`, and destructure where required. If possible, try not to send extra props. If required, send `author`.

Reply:
Okay will change this